source/languages/en/riak/community/product-advisories/appconfig-forward.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Forward Incompatibility of app.config
+project: riak
+version: 1.0.0+ 
+versions: false
+document: reference
+---
+
+Info | Value
+:----|:-----
+Date issued | April 15, 2015
+Product | Riak
+Affected Riak versions | 2.0.0+, 2.1.0+
+
+## Overview
+
+A forward incompatibility of app.config between 1.x and 2.x has been identified.
```

Comment:
Oddity of Basho; I've gotten the feeling that rather than organizing our version numbers `major.minor.patch`, the first two numbers are the major and the last is the minor (`ma.jor.minor`?). This line should probably read  
`A forward incompatibility of app.config between 1.4.x and 2.x has been identified.`

This note applies to the below paragraphs as well.